src/install-protoc.sh

```diff
@@ -28,12 +28,13 @@ get() {
     local url="$1"
     local dest="$2"
 
-    if [[ -w $dest ]]
+    if [[ ! -w $(dirname $dest) ]]
```

Comment:
I had issues with this line calculating sudo correctly. I believe `-w` only evaluates valid paths. Since the destination does not exist, this always returned false. The result was `sudo wget` laid down a file as root:root and chmod below ran without sudo and was unable to change the file bits.